ARrehab/ARrehab/MovementGame.swift

```diff
@@ -15,15 +15,15 @@ Movement Game Entity holds a representatioin of where the user needs to go and t
  */
 class MovementGame : Minigame {
     
-    // Collision group for the MovementTarget
+    /// Collision group for the MovementTarget
     var targetCollisionGroup : CollisionGroup
-    // Collision group for the player.
+    /// Collision group for the player.
```

Comment:
Should use the Player.PLAYER_COLLISION_GROUP class var for consistency

Reply:
#23